### PR TITLE
[RFR] Dicom xlwings

### PR DESCRIPTION
**Pull request by JacobMcAloney**

### CODE REVIEW

**Comment by SimonBiggs** [comment]
@Centrus007 this one was mostly written by me on @JacobMcAloney machine, so if you could give it a once over independent review that would be awesome.

I imagine it won't be simple to test, given you'd need to set up xlwings yourself, but even just a read through, would be helpful.

File: src/pymedphys/xlwings/_level1/dicom.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affero General Public License. These additional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
+
```
Comment (Centrus007) [inline src/pymedphys/xlwings/_level1/dicom.py]: Perhaps include at least a short module docstring explaining main points of the module?

File: src/pymedphys/xlwings/_level1/numpy.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affero General Public License. These additional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
```
Comment (Centrus007) [inline src/pymedphys/xlwings/_level1/numpy.py]: Perhaps include at least a short module docstring explaining main points of the module?

**Comment by Centrus007** [comment]
tests?

**Comment by SimonBiggs** [comment]
Sure can do. Not sure how to include an Excel document within a test...
Have any ideas?

**Comment by Centrus007** [comment]
Probably not worth the testing (for this at least), to be honest. Never used `xlwings` before and was just trying to guess. The gist is that you want to make use of some existing pymedphys functions (e.g. `extract_profiles()`) in an Excel spreadsheet?

I at least spent a bit of time reviewing `xlwings` and everything looks syntactically in order. Pretty happy to merge :)

File: src/pymedphys/xlwings/__init__.py [hunk 3]
```diff
@@ -24,7 +24,8 @@
 # program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
 
 
-"""
+"""Expose a range of pymedphys, scipy, and numpy functions for use within
+Excel via xlwings udf functionality.
```
Comment (SimonBiggs) [inline src/pymedphys/xlwings/__init__.py]: @Centrus007 It's not much, but it gives the gist

**Comment by SimonBiggs** [comment]
@Centrus007 our new registrar is a bit of a gun with Excel, quite new to python though. This is a way for him to hit the ground running in an environment he's comfortable.

Allows him to quickly and easily pull profiles and pdds out of DICOM files within Excel and then do further analysis.

File: src/pymedphys/xlwings/_level1/dicom.py (hunk 1 above)
Comment (SimonBiggs) [reply src/pymedphys/xlwings/_level1/dicom.py]: I just included an xlwings top level module docstring.

**Comment by SimonBiggs** [comment]
@Centrus007 Given there were some unresolved discussion I'll leave this for you to merge when you're happy.

**Pull request merged**